go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/minio/kes
 
-go 1.21
+go 1.22
```

Comment:
Required upgrade because `slog` only supports some functions from v1.22 and later.